package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "test": "npm run lint && nyc ava --verbose --timeout=1m --serial test/*.test.js",
-    "test-no": "nyc ava --verbose --timeout=1m --serial test/*.test.js",
+    "test-no": "nyc ava --verbose --timeout=1m --serial test/pictures.test.js",
```

Comment:
You might want to remove this one

Reply:
Oups, I did not see that.
Ok I made the change.